src/plugins/mer/mertarget.cpp

```diff
@@ -148,7 +162,8 @@ QVariantMap MerTarget::toMap() const
 
 bool MerTarget::isValid() const
 {
-    return m_sdk && !m_name.isEmpty() && !m_qmakeQuery.isEmpty() && !m_gccMachineDump.isEmpty()
+    return !m_name.isEmpty() && !m_qmakeQuery.isEmpty() && !m_gccMachineDump.isEmpty()
```

Comment:
I don't like the fact that you dropped m_sdk check here. Sure, it should never be null, but I would like to make sure, as programmers are known to make mistakes. I would consider even adding an assert in constructor.

Reply:
Me neither - it's a mistake I did while rebasing, thanks :)